src/towncrier/build.py

```diff
@@ -67,9 +81,18 @@ def _get_date() -> str:
 @click.option(
     "--yes",
     "answer_yes",
-    default=False,
+    default=None,
     flag_value=True,
     help="Do not ask for confirmation to remove news fragments.",
+    callback=validate_answer,
+)
+@click.option(
+    "--keep",
+    "answer_keep",
+    default=None,
+    flag_value=True,
+    help="Keep the newsfragments.",
```

Comment:
Can we have more help text here?

In an ideal case, the whole info from the "cli.rst" file would be placed here.

In this way, we can use cli-man to generate a man version of the docs...and we can also use `towncrier build --help` to get that info.

these days, when I use a CLI, in order to get some documentation, I first try my luck with "--help" and only if it fails I use "man" or the project website.